modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -14617,6 +14769,8 @@ terms-of-use=Terms of Use
 terms-of-use-web-content=Terms of Use Web Content
 terms-of-use-web-content-help=Specify the group ID and the article ID of the web content that will be displayed as the terms of use. The default text will be used if no web content is specified.
 tertiary=Tertiary
+tertiary-horizontal=Tertiary Horizontal
+tertiary-vertical=Tertiary Vertical
```

Comment:
For future, just add all the Language keys to the end of the file. We can let the buildLang task sort them for us. This way there are fewer merge conflicts when rebasing.